fast64_internal/oot/scene/exporter/to_c/scene_table_c.py

```diff
@@ -1,283 +1,257 @@
-import os, bpy
+import os
+import bpy
+
+from dataclasses import dataclass, field
+from typing import Optional
 from .....utility import PluginError, writeFile
 from ....oot_constants import ootEnumSceneID, ootSceneNameToID
 from ....oot_utility import getCustomProperty, ExportInfo
+from ....oot_level_classes import OOTScene
+
+
+@dataclass(unsafe_hash=True)
+class SceneTableEntry:
+    index: int
+    original: str
+    scene: Optional[OOTScene] = None
+    exportName: Optional[str] = None
+    prefix: Optional[str] = None  # ifdefs, endifs, comments etc, everything before the current entry
+    suffix: Optional[str] = None  # remaining data after the last entry
+    parsed: Optional[str] = None
+
+    specName: Optional[str] = None
+    titleCardName: Optional[str] = None
+    enumValue: Optional[str] = None
+    drawConfigIdx: Optional[str] = None
+    unk1: Optional[str] = None
+    unk2: Optional[str] = None
+
+    def __post_init__(self):
+        self.paramStart = "DEFINE_SCENE("
```

Comment:
```suggestion
        macroStart = "DEFINE_SCENE("
```
seems like a better name?
and it can be just a variable